app/src/controllers/launch/actionValidators.js

```diff
@@ -6,7 +6,7 @@ const isAdminOrProjectManager = (state) => {
   const activeProject = activeProjectSelector(state);
   return (
     isAdminSelector(state) ||
-    user.assigned_projects[activeProject].projectRole !== 'PROJECT_MANAGER'
+    user.assigned_projects[activeProject].projectRole === 'PROJECT_MANAGER'
```

Comment:
We have common constants for project roles. Is there possibility exists to use permission_resolver created by Yana?